_posts/2018-07-31-Agile-Leadership-Game.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+authors: [maryse_meinen]
+title: "Agile Leadership Game"
+image: /img/agile-leadership-game/How-Agile-Are-You.jpg
+tags: [Agile, Agile Leadership Game, Agile Consortium]
+category: Agile
+comments: true
+---
+
+Last week we, as the guild of agile coaches from Ordina, tried and tested the Agile Leadership Game developed by the Agile Consortium called 'How Agile Are You?'.
+In this blog post, we will provide you with some more information about the game.
+
+### Details
+* 2-6 players (even though more is possible)
+* Intended audience is management
+* Takes 1.5 hours (on average)
+* Needs a skilled facilitator
+
+### Game Play
+This game has been designed to get discussions on agile leadership going and to ensure that people start observing behaviour that belongs (or does not belong) to an agile organisation.
+It is specifically intended for management teams.
+The game is focused on agile mindset and agile leadership: what do people and organisations need from management when engaged in an agile transformation?
+
+Participants are asked to very rapidly (! ten seconds per card on average :) !) divide forty two cards with 'characteristics' of organisations, over one of the quadrants on the board.
```

Comment:
I think fourty-two needs a hyphen? (also fourty instead of forty as the article is written in British-English :))